### ISSUE TITLE
[Feature Request] Treeview Component branch lines

### ISSUE THREAD

**Issue by simonadler1** [vuetifyjs/vuetify#11506]
### Problem to solve
the tree nodes hierarchy is visually hard to follow when dealing with a large tree. 

### Proposed solution
Branch lines for making the tree nodes hierarchy easier to follow.

**Comment by gilad7** [comment]
For anyone looking for a quick solution in the meantime I added this to my scoped style tag:

`::v-deep .v-list-group__items {
  border-left: 2px solid rgba(var(--v-theme-onSurface), 0.5); /* Green border for the group items */
  margin-left: 12px;
}

::v-deep .v-list-group--prepend {
  --parent-padding: 0;
  padding-left: 16px;
}

::v-deep .v-list-item--density-compact:not(.v-list-item--nav).v-list-item--one-line {
  padding-inline: 0px;
}`

**Comment by johnleider** [comment]
I'd still like to do this but am not sure how to begin to approach it. Adding help wanted in case someone has an idea.

**Comment by J-Sek** [comment]
- [only gridlines](https://play.vuetifyjs.com/#eNrVV01P5DgQ/SulSKuGUbu708AMk2WQ2J3LXpYLmj0MCDlJNbHk2FnHCSDEf99yvpzQTYNm5zIcIHG9V6l6VXaZ709BaZLlRVEs6gqDKDizmBeSWzy/VgBnNeNF0Tz2LyzmplsYLzErrMTzb1cGsRZ4D/fCZiCFwvJs+RI1opeESwjIY5RfroOGcB1AJlJkKVouZAk1y3XamFHxWGJKgGUflHfuw8y5UOOPJFqRJ4UGcv7A7kVqM3J2slpdBwOsAdouer8IECWSlyXhn2DmkmJNjLMIumDg+TqY4AVJ6PDN36nNLbGaywqdnfIYG3WBinEp/VqfZJvmkMUo9SHTTgd6PluOSkivZWJEYaFEW7WVFHmhjYUnMLiBZ9gYncOMqj9zRvpIaYfUvjjMgTUVHnpjkxeZvrdhPPUxijSCcN6/NZWOYEatJUXCrSAuRLPBnmRCpgZV1PtpfDVO1vOB/SeXqFJuIALKbgbPA78HH43AGaWCr0OPPfQfjEurTb4Nvume+pVJeidb6X3VSZWjsu/IbdII5OzjKEDvkAphxeZx7O5Vl1tue9efJtyxe9rtL13vde/VO/XqCZXiA0lny6nMU8JnT4i1tqU1vHiV1Mve/0wRY+ukri81DVc7Rc1pNxjB5fr/yRr69v55uoajdq9ZbNWbwoajnqdjgZv0bcqo92s6AVWq739qLfZum3DXvrlXUvP0vWdC+NEncJlYHdNhHkGRbnZt9PCTB/+ta8z3o0edfVXRqUCNQujM5vLdB0P4eSvDbzTC9A8cC+vdLdxHNvG4v4Vbd6HP7g9eioRm7aOu3IEFeXG8Xf+ONurKi7TmKqFxYDHJlPi3wrfIowa9kBJ4TB90By3RUmFe0l7d3a2zUev+pazR7tu63lXJ9cloGGi1QZIlQRCOlVaJm0HuwK/F63W9cQO0HZnd+LSP3Y1l4e8HC38R6AvImDc3lvaeQUEVD7+3kOUHuLr8ehlBzamSNGBL2GjTx1ETq7TMTVfIUNxllibWafGww854m8s5LboDo4ev3wd3fwXNcs87Lh7gw7KlUpoDM4qRIkTfpLfuEkLjjgSedVkBFLoU7gMkbVxqWVkcTFYXNDVXvw0LEjdEXg3vsbZW59MVk6Jhpg0t5WVGnWfuYn5Ash00MmeYI9OKlZXZ8AQP57BarA9feuj0H1jbxfGUgqepUHdMqMZaWm7o40JlNDZsh3qe6nNndFXc3ra3oUEgr4VBuoOJutdiD3tL5R8ROeEyOWAhfIAD1zWwhDWwtrL0eOhTbSvg4fv0aai/Rq2ahFq+u54oyzrcoZOhU6R30hSDftFW77Z38DwPjhYni/AocA+ni+NgvqHTFueTf5DatZv/AEzUt1E=)
- [gridlines with lines to leafs](https://play.vuetifyjs.com/#eNqtWFtv2zYU/iuEh81Wayq2m7Sd16bo1pe+LC9FMaAJClqibLYUqVGU7CDIf98hRUqULdu9LA+xdC4fz+UjeexPD6NSJRdviyKuKzpajl5pmhecaHp9KxB6VWNSFPbRv+AVUU4QirBmmtPrjx8UpTWjW7RleoM4E7R8dbFvFbiXYJeAIVlR/vp2xBIpytsR2rCU4pRqwniJapzL1KrLjdy+dyYXJ2DsusdhqCArTtMAJIixyzYnTISLwMKAJKhCOdnhLUv1BsCuZrPbUWtmDbUrQidEKKWiZPoeHBKZFyTRt6NQvUw4KUvQPqCxKR22KYyXyMWKHvfsGTTK2NvPvs6IcE14RY0e0gyVsqACE847WRA7RO/7D+Vi2V7JfykUBffUhmlWmSJW3oDERNeDsWUwzYRA4T84rKk2MJPQLeq1selDj39e2JW06wcI24YEXWyb5loKzz1QeC0TxQqNSqqrhtssL6TS6AEpmqFHlCmZozHsh7FRmtx124bXxmaiVUWjTtnWqK/uDDLGqVE+NHFudM6XaJynDHMi1hVZU2xkV+NpY/CldGoBjP1SerH24tZL3xe0SadzlcJZJeCMzbvX5em+f07U11RuW4sizZyJCRnDK17JXasV61DLcoBoY9vpUJfKpMqp0FhW2lDZm+24z8Ga0V1CeRuerENdDZtX7vuTmp21yYvLMzaPTXOySiSaAUkHyem7xTJkFXGyYTxVRqOAOkp48r9xi0meUoXNBhsjH4CVWR4h72XJ8Mki2uMwLgvO9GQcj6OY6AmeR3fojcc0CfjQXeSeVHbnA6k+NeguWogXunzlSgF9MUtANO9cQ1oyAYzLZ+khejAe6nlrH8LB7tAsu+/AjgIegHrgFz3fEByupD7wHjgcPuD/cto6MJHSXQy5oce7PcfH3nuoDTUHWc9ng2nncIgoRvjiZxKfz384830VrNBALrpi1HilRVOMA6jW/llonxCVnnO4DB22UHC5PeLyIx3wUi/pkXk+xOat4JKkZ9ncADzvgr9JtFxRFcPR1g/emb7oTP+WNc1P2QYM/FBpaYgRm3M8ND6d2e8HmX00p9V3b9LFMF19VAHeabo2YPMurz9JyRKYq+7hECpjOFkPO+6cAga+TWsiErgtNU02gv1b0dOuARnfco7IChZDcHfHKVP7Tkd3cAMV0PS90ErGcKkM9W5x1Rn+JUVGoRwJRcz4pJW9FWK4a4538s7MFc3V66YKfe9G27gbV+JulvNNw7hTW00zSQIZit0fx0wkTIwwPS7RLL4KjDgrNbbj3oay9QZu4MsZoKCLJ3BH2CnTS1KakYqb15fFDj25GFgIrCFrH80C7AbDoSTrhZ0QnkxqoiZHwSL0FC2KXeTgILgPN+9ulgicmJmqSpRJ5ascJtWAoGsQwpGGfI4mtG8wN592BG39LoPUoUnWc61kVXz+3Fym163YvHfb7LflikKMNNx4ZviE+xT4M3aJ2RFJwpQPUSyBxaXklaaB0gdyUK2unr5I5o/TDIxtfcMdcMobXaAFwocm/XaAUQcYrriSWsscxofZr6GUJF9NmQTsG7MWfI9bK5IyyL4XmJZImQR7+1WtV6Sjx4bmQGaBy0plJKHR9Eg2ju9R9H9i9c+eQ+M+s/tLa0VEWRA4JfR5z+HiFiRNmVhjJqxpqYmC9jKxgYlCB3Z2KWAb9ME+mi8u/0zms9mvHZodA9uPQS63VO0oqShgsdpTcsg72AIh2gH/h9h/gvtaFo7JeI6eoEnbxf0TzDPY7lV4jKIu6WA/WJRzG6Hz9LyeBRJlhnXL1yWCaOF7ctuBn+TZ/iLuoPymXT9Mkr0zoEEyQy98wXIe0Xdte7+gowAcyb2uL+HHCI3tmHCGEIPH6VJIPVluSDl52ldE0bfyy3esd7sMkqWXCvyDW9ndxKPH6ehZfBXPn43Mw8v4cjTNYBii096PXo3s7j+ixakX)
